### PR TITLE
fix: added logs related to executing commands in the container

### PR DESCRIPTION
**Pull request by sxllwx**
Signed-off-by: scott <scottwangsxll@gmail.com>

Fixes：

In the process of troubleshooting #10490, we found that our `wait` received more than one sigterm.

However, when wf-controller sends sigterm for the second time (actually through argoexec kill), it may cause container-crash (signal processing is asynchronous), and we need this part of the log to support our troubleshooting.

Please do not open a pull request until you have checked ALL of these:

* [x] Create the PR as draft .
* [ ] Run `make pre-commit -B` to fix codegen and lint problems.
* [x] Sign-off your commits (otherwise the DCO check will fail).
* [x] Use [a conventional commit message](https://www.conventionalcommits.org/en/v1.0.0/) (otherwise the commit message check will fail).
* [x] "Fixes #" is in both the PR title (for release notes) and this description (to automatically link and close the issue).
* [x] Add unit or e2e tests. Say how you tested your changes. If you changed the UI, attach screenshots.
* [x] Github checks are green.
* [x] Once required tests have passed, mark your PR "Ready for review".

If changes were requested, and you've made them, dismiss the review to get it reviewed again.

### CODE REVIEW

**Review by tooptoop4 — CHANGES_REQUESTED** [review]
Use Debug level instead of Info level (already the amount of logs are expensive). Could the command contain sensitive text (ie result of secret/pw)?

File: workflow/signal/signal.go [hunk 1]
```diff
@@ -45,6 +45,13 @@ func SignalContainer(restConfig *rest.Config, pod *corev1.Pod, container string,
 
 func ExecPodContainerAndGetOutput(restConfig *rest.Config, namespace string, pod string, container string, command ...string) error {
 	x, err := common.ExecPodContainer(restConfig, namespace, pod, container, true, true, command...)
+	log.
+		WithField("namespace", namespace).
+		WithField("pod", pod).
+		WithField("container", container).
+		WithField("command", command).
+		WithError(err).
+		Info("exec command")
```
Comment (alexec) [inline workflow/signal/signal.go:54]:
```suggestion
		Debug("exec command")
```

**Review by alexec — CHANGES_REQUESTED** [review]
Do we need this if #10523 works?

File: workflow/signal/signal.go (hunk 1 above)
Comment (sxllwx) [reply workflow/signal/signal.go:54]: > Do we need this if #10523 works?

I think it is needed, it can provide a complete error context.

Comment (sxllwx) [reply workflow/signal/signal.go]: fixed, PTAL. 😄

File: workflow/signal/signal.go [hunk 2]
```diff
@@ -45,6 +45,13 @@ func SignalContainer(restConfig *rest.Config, pod *corev1.Pod, container string,
 
 func ExecPodContainerAndGetOutput(restConfig *rest.Config, namespace string, pod string, container string, command ...string) error {
 	x, err := common.ExecPodContainer(restConfig, namespace, pod, container, true, true, command...)
+	log.
```
Comment (alexec) [inline workflow/signal/signal.go:48]: do you want to move logging into `ExecPodContainer`?

Comment (sxllwx) [reply workflow/signal/signal.go]: ok~, PTAL, thx

**Review by alexec — APPROVED** [review]

**Pull request merged**